pkg/stack/unwind/compact_unwind_table_test.go

```diff
@@ -410,6 +411,50 @@ func TestCompactUnwindTableArm64(t *testing.T) {
 	}
 }
 
+func ensureSorted(t *testing.T, ut CompactUnwindTable) {
+	t.Helper()
+	lastPc := uint64(0)
+	for _, row := range ut {
+		require.Greater(t, row.Pc(), lastPc)
+		lastPc = row.Pc()
+	}
+}
+
+func ensureNoDuplicatedPc(t *testing.T, ut CompactUnwindTable) {
```

Comment:
nit
```suggestion
func ensureNoDuplicatedPC(t *testing.T, ut CompactUnwindTable) {
```

Reply:
This makes sure we don't have duplicates in a row, right?
So the input should be sorted if we want to make sure there are no duplicates globally, no?

Reply:
We check these two invariants already, both that they have to be sorted by PC and that there should not be any duplicates. See TestIsSorted